src/main/java/com/sumindaa/demo/employeeservice/controller/EmployeeController.java

```diff
@@ -10,4 +11,22 @@ public String getAllTopics(){
         return "all topics";
 
     }
+    @RequestMapping ("/employees/{id}")
+    public Employee getEmployee(@PathVariable int id){
+        return EmployeeService.getEmployee(id);
+    }
+@RequestMapping (method = RequestMethod.POST, value = "/employees")
+    public void addEmployee(@RequestBody Employee employee){
```

Comment:
Could you able to access EmployeeService this way? 
EmployeeService.getEmployee(id);

Is this compiling?

Reply:
Done. I made them static

Reply:
Can you do a mvn clean install and confirm everything is OK, 
AND this is very important, before every push to origin. You can understand why?